tests/test_cv.py

```diff
@@ -0,0 +1,19 @@
+import scanpy as sc
+
+from augurpy.cv import run_cross_validation
+from augurpy.estimators import create_estimator
+from augurpy.read_load import load
+
+adata = sc.read_h5ad("tests/sc_sim.h5ad")
+adata = load(adata)
+
+# subsample 100 obs
+sc.pp.subsample(adata, n_obs=100)
+
+rf_estimator = create_estimator("random_forest_classifier")
+
+
+def test_run_cross_validation():
+    """Test run cross validation with categorical and classifier."""
+    cv = run_cross_validation(adata, rf_estimator, subsample_idx=1, folds=3)
+    assert cv["test_auc"] is not None
```

Comment:
Maybe we can compare this against an exact value in the future, but think for now this suffices.

Reply:
Ah I was about to explain that these are difficult to compare because the values fluctuate / the random seeds are different each time - should I set these somewhere to make the results reproducible? Normally the user sets them though, right?

Reply:
Yeah, the users sets them. But you can fix them as well :)

Reply:
Here for the test case.